### ISSUE TITLE
Error on boot - wine.log

### ISSUE THREAD

**Issue by Steveiwonder** [BitR/empyrion-docker#16]
Hey,

I had this all working on one machine, started again on another new machine, dedicated server running AMD instead of Intel and now I get this error in the wine.log - Would anyone know how to solve?

```
000b:fixme:winediag:__wine_start_process Wine Staging 5.7 is a testing version containing experimental patches.
000b:fixme:winediag:__wine_start_process Please mention your exact version when filing bug reports on winehq.org.
0010:err:ole:StdMarshalImpl_MarshalInterface Failed to create ifstub, hres=0x80004002
0010:err:ole:CoMarshalInterface Failed to marshal the interface {6d5140c1-7436-11ce-8034-00aa006009fa}, 80004002
0010:err:ole:get_local_server_stream Failed: 80004002
0013:err:module:LdrInitializeThunk "UnityPlayer.dll" failed to initialize, aborting
0013:err:module:LdrInitializeThunk Initializing dlls for L"Z:\\home\\user\\Steam\\steamapps\\common\\Empyrion - Dedicated Server\\DedicatedServer\\EmpyrionDedicated.exe" failed, status c0000005
```

**Comment by BitR** [comment]
Did you start it up in a clean gamedir? And do you have any other logs available?

**Comment by Steveiwonder** [comment]
Heya @BitR - I don't have any other log files except some steam ones, but it's coming from the dedicated server so there shouldn't be anything in those. I have looked, I've installed it into a fresh directory several times over without any success. 

I found this https://forum.winehq.org/viewtopic.php?f=8&t=35840 but I can see in the docker file you already have the correct architecture installed.

I have a dedicated empty 30GB ssd just for this server too.

If it helps, I can get the server to run fine outside of a docker container.

I appreciate your help.
